frontend/src/views/ImageDetailsPage/components/TextTile/TextTile.styles.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const Template = styled.article`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 2%;
+  max-height: 260px;
+  overflow-x: hidden;
+  overflow-y: auto;
+`;
```

Comment:
Maybe rename that styled component and move to global styles? I see you use two styles for tiles, so they can be reusable from global style. 🤔